tools/state-viewer/src/cli.rs

```diff
@@ -721,6 +725,22 @@ impl ViewChainCmd {
     }
 }
 
+#[derive(clap::Parser)]
+pub struct ViewGenesisCmd {
+    /// If true, displays the genesis block built from the config.
+    #[clap(long)]
+    view_config: bool,
```

Comment:
nit: I think `config` is a bit confusing, for me the first thought would be config.json. How about replacing it with `file` and mention that this is about `genesis.json` in the comment? 

Side note - the path and name of the genesis file is actually defined in config so it may be different than `genesis.json`.